src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,52 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
```

Comment:
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
    
    Animal.alive.push(this);
```
health is not configurable, 100 by default for all animals